yoyodyne/train.py

```diff
@@ -341,23 +339,23 @@ def add_argparse_args(parser: argparse.ArgumentParser) -> None:
         "--patience",
         type=int,
         help="Number of epochs with no progress (according to "
-        "`--patience_metric`) before triggering early stopping. "
-        "Default: not enabled.",
+        "`--patience_metric`) before triggering early stopping.",
     )
     parser.add_argument(
         "--patience_metric",
         choices=["accuracy", "loss", "ser"],
         default=defaults.PATIENCE_METRIC,
-        help="Stops early when validation `accuracy` stops increasing or "
-        "when validation `loss` or `ser` stops decreasing. "
-        "Default: %(default)s.",
+        help="Stops early when validation `accuracy` does not increase or "
+        "when validation `loss` or `ser` does not decrease for `--patience` "
+        "epochs. Default: %(default)s.",
```

Comment:
I am confused about the final "s" in the string here.

Reply:
Patience is a number, so after 5 (say) epochs...

Reply:
I removed it though, I decided it is awkward as you say.